packages/wrangler/src/deploy/deploy.ts

```diff
@@ -570,19 +572,6 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			dependencies[modulePath] = { bytesInOutput };
 		}
 
-		// Add modules to dependencies for size warning
-		for (const module of modules) {
-			const modulePath =
-				module.filePath === undefined
-					? module.name
-					: path.relative("", module.filePath);
-			const bytesInOutput =
-				typeof module.content === "string"
-					? Buffer.byteLength(module.content)
-					: module.content.byteLength;
-			dependencies[modulePath] = { bytesInOutput };
-		}
-
```

Comment:
Is this deletion intentional?

Reply:
Yes, this block of code was duplicated—probably from a bad rebase/merge?